scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -1088,7 +1088,7 @@ def lnet_self_ping(self):
             return False
     return True
 
-def update_motr_hare_keys_for_all_nodes(self, k8=False):
```

Comment:
Changes  for function 'update_motr_hare_keys_for_all_nodes' are to remove codecy errors.